components/sensor/emontx.rst

```diff
@@ -0,0 +1,308 @@
+OpenEnergyMonitor EmonTx Sensors
+================================
+
+.. seo:: :description: Instructions for setting up OpenEnergyMonitor EmonTx energy monitors with ESPHome. :image: emontx.jpg :keywords: EmonTx, OpenEnergyMonitor, energy monitor, power monitoring, CT clamp
+
+.. _emontx-component:
+
+Component/Hub
+-------------
+
+The ``emontx`` component allows you to use an EmonTx device with ESPHome.
+This component is a global hub that establishes the connection to the EmonTx via :ref:`UART <uart>` and translates the received data.
+Using the :ref:`emontx sensors <emontx-sensors>`, you can then create sensors for Home Assistant that track voltage, current, power, and temperature readings from the EmonTx.
+
```

Comment:
The component can send data to an MQTT Broker either as JSON or as individual topics to be consumed by any system.